airbyte-featureflag/src/test/kotlin/ClientTest.kt

```diff
@@ -0,0 +1,229 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+import com.launchdarkly.sdk.LDUser
+import com.launchdarkly.sdk.server.LDClient
+import io.airbyte.featureflag.*
+import io.mockk.*
```

Comment:
Nit:  avoid * imports.

Reply:
#21136 
I updated our `code style` docs to set this for Kotlin correctly